_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -53,9 +53,23 @@ Once EOL is reached, you will need to upgrade to a newer version.
 
 _TBA: to be announced_
 
-## Foot notes
+## Protocol versioning
 
-Bitcoin Core software versioning is similar, but not identical to [SemVer][]. Bitcoin Core implements a full stack of wallet, full node, networking and consensus rule validation. However, versioning of the Bitcoin system is much more complex; there are block versions, transaction versions, p2p versions, and consensus rules are all versioned separately, and the consensus rule. [SemVer][] is not designed to handle this complexity and while Bitcoin Core adheres in the most part to the spirit of [SemVer][], there are notable exceptions, especially for consensus rules proposals.
+The description above only describes Bitcoin Core software releases. Many other parts of the Bitcoin system contain their own versions.  A few examples:
+
+- Every **block** and **transaction** contains a version number.
+- The **P2P network protocol** uses version numbers to allow nodes to announce what features they support.
+- Bitcoin Core's **built-in wallet** has its own internal version number.
+
+These versions numbers are deliberately decoupled from Bitcoin Core's version number as the Bitcoin Core project either has no direct control over them (as is the case with blocks and transactions), or tries to maintain compatibility with other projects (as is the case with the network protocol), or allows for the possibility that no major changes will be made in some releases (as is sometimes the case with the built-in wallet).
```

Comment:
Is that actually used?  I can't think of any release where I had to explicitly opt-in to a wallet change; I just made a backup before switching to the new version, started the new version, and it worked.

Moreover, I don't know of any way to check the wallet version without starting bitcoind/bitcoin-qt (since you have to run an RPC to get the wallet version number).